src/Microsoft.OData.Core/UriParser/Binders/SelectExpandBinder.cs

```diff
@@ -964,15 +979,39 @@ private static void VerifySelectedPath(SelectTermToken selectedToken)
                     throw new ODataException(Error.Format(SRResources.SelectExpandBinder_SystemTokenInSelect, current.Identifier));
                 }
 
+                // Be design, the "/$count" should be built using SystemToken, but now, ODL is using NonSystemToken.
+                // Shall we support "no-dollar sign"?

```

Comment:
/$count requires the '$', but ($count=true) does not.

Reply:
Cool.